packer/builders/google-cloud/image-ubuntu/provisioner.json

```diff
@@ -0,0 +1,26 @@
+{
+  "builders": [
+    {
+      "type": "googlecompute",
+      "account_file": "../account.json",
+      "project_id": "blackops-qa",
```

Comment:
is it OK to have this hardcoded here ? (I'm not even sure what is this JSON)

Reply:
I'm thinking in a better way to store this, but at this moment we are storing this file `account.json` inside on container as variable and writing in this file to use to access the provider resources.